elixir/apps/api/lib/api/device/channel.ex

```diff
@@ -85,6 +85,7 @@ defmodule API.Device.Channel do
     connected_gateway_ids = Map.get(attrs, "connected_gateway_ids", [])
 
     with {:ok, resource} <- Resources.fetch_resource_by_id(resource_id, socket.assigns.subject),
+         # TODO:
          # :ok = Resource.authorize(resource, socket.assigns.subject),
```

Comment:
@bmanifold if you feel like it - you can also handle policy evaluation when connecting to a resource.